code/AdobeEdge.brs

```diff
@@ -180,26 +180,52 @@ function AdobeSDKInit() as object
         ' If you would like to include this information in your dataset, add the "Implementation Details"
         ' field group to the schema tied to your dataset.
         '
+        ' This function allows passing custom identifiers using identityMap.
+        '
         ' @param data as object : xdm data
         ' @param [optional] callback as function(context, result) : handle Edge response
         ' @param [optional] context as dynamic : context to be passed to the callback function
         '
         ' Example 1:
         '
+        ' customIdentityMap = {
```

Comment:
I think this API doc became too long, what do you think about covering it under documentation?

Reply:
I think it's better to have samples under docs and remove from here. @yangyansong-adbe do you agree?

Reply:
Either way. If we want to cover examples on the doc page, let's remove all examples from the API doc.